docs/main.js

```diff
@@ -105,7 +105,7 @@
     var versionParam = version ? '?v=' + version : '';
     var code = event.target.nextElementSibling.textContent;
     var encoded = fixedEncodeURIComponent(code);
-    location.assign(location.origin + '/repl/' +
+    window.open(location.origin + '/repl/' +
       versionParam + '#;' + encoded);
```

Comment:
since we are popping a new tab, should we include a window name (second param) so they all wind up in the same new window? Or is it preferable to keep spawning new ones?